src/main/kotlin/com/terraformation/backend/tracking/api/SubstrataController.kt

```diff
@@ -53,21 +57,54 @@ class PlantingSubzonesController(
 
     return SimpleSuccessResponsePayload()
   }
+
+  @Operation(summary = "Updates the planting-completed state of a substratum.")
+  @PutMapping("/substrata/{id}")
+  fun updateSubstrata(
+      @PathVariable id: PlantingSubzoneId,
+      @RequestBody payload: UpdateSubstratumRequestPayload,
+  ): SimpleSuccessResponsePayload {
+    plantingSiteStore.updatePlantingSubzoneCompleted(id, payload.plantingCompleted)
+
```

Comment:
The old endpoint should only be needed until the FE uses the new one, because the mobile app doesn't use this. I was unable to get OpenAPI to allow a request payload that works for either name while still keeping the old name in the generated schema in the FE, which was necessary because the FE references it directly using `components['schemas']['UpdatePlantingSubzoneRequestPayload']`.